docs/backend_architecture/upgrade.rst

```diff
@@ -20,6 +20,12 @@ that take a ``choices`` keyword argument, where the choices are strings. The
 strings should have no prefix (``u`` or ``b``) and the migration should contain
 ``from __future__ import unicode_literals`` as an import.
 
+For any migrations involving Foreign keys to the FacilityUser model, the migration
+will automatically be generated to use a swappable dependency on settings.AUTH_USER_MODEL.
+This should be changed to remove the swappable dependency, set a fixed migration dependency
+and remove the AUTH_USER_MODEL setting from the settings file. FacilityUser should be used
+instead.
```

Comment:
I don't know if this is really better, but more as a brainstorming idea than a specific suggestion, what if this
> This should be changed to remove the swappable dependency, set a fixed migration dependency
and remove the AUTH_USER_MODEL setting from the settings file. FacilityUser should be used
instead.

becomes: 

> Linting checks intentionally remove the swappable dependency, set a fixed migration dependency, and remove the AUTH_USER_MODEL setting from the settings file. FacilityUser is used instead [of.... ? I do not actually know. AUTH_USER_MODEL?]. We do this so that [I do not know this either, something something... KDP]